src/test/java/net/sf/jabref/logic/importer/fetcher/MrDLibFetcherTest.java

```diff
@@ -0,0 +1,40 @@
+package net.sf.jabref.logic.importer.fetcher;
+
+import java.util.List;
+
+import net.sf.jabref.logic.importer.FetcherException;
+import net.sf.jabref.model.entry.BibEntry;
+import net.sf.jabref.model.entry.FieldName;
+import net.sf.jabref.testutils.category.FetcherTests;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import static org.junit.Assert.assertEquals;
+
+@Category(FetcherTests.class)
+public class MrDLibFetcherTest {
+
+    private MrDLibFetcher fetcher;
+    private BibEntry bibEntry;
+
+    @Before
+    public void setUp() {
+        fetcher = new MrDLibFetcher("", "");
+        bibEntry = new BibEntry();
+        bibEntry.setField(FieldName.TITLE, "lernen");
+    }
+
+    @Test
+    public void testPerformSearch() throws FetcherException {
+        List<BibEntry> bibEntrys = fetcher.performSearch(bibEntry);
+        assertEquals(false, bibEntrys.isEmpty());
```

Comment:
Please also create a functional test...i.e. that a proper list of bibentries is returned.
Also have a look at https://github.com/JabRef/jabref/wiki/Code-Howtos#test-cases and make sure that the tests adhere to these best-practices.